lib/dai-plugin-mcd/test/CdpTypeService.spec.js

```diff
@@ -5,21 +5,23 @@ import { REP, DGX } from '../src';
 
 let service;
 
-describe.each([ETH, REP, DGX])('ETH, REP, DGX', collateral => {
+const scenarios = [ETH, REP, DGX].map(c => [c.symbol, c]);
+
+describe.each(scenarios)('%s', (symbol, collateral) => {
```

Comment:
@tyler17 note my change here, to have the argument to `each` be an array of arrays. this produces better output (see below) and is how the method is [designed to be used](https://jestjs.io/docs/en/api.html#1-describeeachtable-name-fn-timeout). please change the auction tests to be this way as well.

```
 PASS  lib/dai-plugin-mcd/test/CdpTypeService.spec.js
  ✓ list all cdp type objects (3ms)
  ETH
    ✓ get total collateral (41ms)
    ✓ get total collateral in USD (66ms)
    ✓ throw in get total collateral (35ms)
    ✓ get total debt (20ms)
    ✓ get debt ceiling (19ms)
    ✓ get liquidation ratio (26ms)
    ✓ get price (26ms)
    ✓ get liquidation penalty (28ms)
    ✓ get annual stability fee (24ms)
    ✓ get ilk id (1ms)
  REP
    ✓ get total collateral (25ms)
    ✓ get total collateral in USD (37ms)
    ✓ throw in get total collateral (19ms)
    ✓ get total debt (20ms)
    ✓ get debt ceiling (17ms)
    ✓ get liquidation ratio (20ms)
    ✓ get price (15ms)
    ✓ get liquidation penalty (20ms)
    ✓ get annual stability fee (17ms)
    ✓ get ilk id (1ms)
  DGX
    ✓ get total collateral (32ms)
    ✓ get total collateral in USD (36ms)
    ✓ throw in get total collateral (15ms)
    ✓ get total debt (19ms)
    ✓ get debt ceiling (15ms)
    ✓ get liquidation ratio (19ms)
    ✓ get price (11ms)
    ✓ get liquidation penalty (23ms)
    ✓ get annual stability fee (34ms)
    ✓ get ilk id
```

Reply:
Good point, I pushed an update to the auction tests